src/engine/Settings.cs

```diff
@@ -367,7 +415,7 @@ private static Settings LoadSettings()
 
             if (error != Error.Ok)
             {
-                GD.Print("Settings configuration file is missing or unreadable, using default settings.");
+                GD.Print("Failed to load settings configuration file, using default settings.");
```

Comment:
Shouldn't this error stay the same?

Reply:
Well I changed it cause it only fails if the file fails to open, it's not really reading it. So it'd either have to be locked or like, so corrupted that the open would fail, no? I thought calling the file unreadable made more sense in context of the deserialize.

Reply:
The wording `load` to me indicates that some processing attempted to happen. It's clearer to explicitly say that the file could not be read at all.

Reply:
Well it'd probably be more proper to say something like "Failed to open settings configuration file for reading, using default settings instead."

Reply:
I think that the original wording did convey that message pretty well: `file is missing or unreadable`